test/integration/patients/patient/sidebar.js

```diff
@@ -68,7 +70,7 @@ context('patient sidebar', function() {
       .as('patientSidebar')
       .should('contain', 'First Last')
       .should('contain', formatDate(dob, 'LONG'))
-      .should('contain', `Age ${ moment().diff(dob, 'years') }`);
+      .should('contain', `Age ${ moment(testDateSubtract()).diff(dob, 'years') }`);
```

Comment:
I think this'd just be `testDate()`

Reply:
It's interesting that this works though...  _crazy_ thought.. particularly since we you just did all of this..  _but_ we could support all the same functionality with `testDate()` for now `testDate(1)` for add one day and `testDate(-1)` or subtract one day..  Do you think it's better that we have more functions in the case, or is it faster if we have less api to manage?  Generally I'd say the more explicit API is better, but there's a case that test helpers should be easy to use and malleable. 
But for instance our current API could be written:
```js
function testDate() {
  return testDateAdd();
}

function testDateAdd(time, key = 'days') {
  return moment(date).add(time, key).format('YYYY-MM-DD');
}

function testDateSubtract(time, key = 'days') {
  return testDateAdd(-time, key);
}
```

Reply:
Hmm. Normally I think I'd go for consolidating the API like that. But I think in this case, being explicit makes the tests more readable and I'd prefer that.

Reply:
good enough for me